src/ApiService/ApiService/onefuzzlib/ProxyOperations.cs

```diff
@@ -213,7 +213,7 @@ public ProxyOperations(ILogTracer log, IOnefuzzContext context)
 
     private async System.Threading.Tasks.Task<Proxy> SetProvisionFailed(Proxy proxy, VirtualMachineInstanceView? instanceView) {
         var errors = GetErrors(proxy, instanceView).ToArray();
```

Comment:
NIT: Don't need `ToArray` if you're going to `ToList` after
```suggestion
        var errors = GetErrors(proxy, instanceView);
```